package.json

```diff
@@ -35,19 +35,19 @@
     "@types/lodash": "^4.14.149",
     "@types/node": "^8.10.59",
     "@types/optimist": "0.0.29",
-    "@types/prettier": "^1.18.3",
+    "@types/prettier": "^1.19.1",
     "coffee-script": "^1.10.0",
     "coffeelint": "^1.15.0",
     "coffeescope2": "^0.4.5",
     "depcheck": "^0.6.7",
     "glob": "^7.1.6",
     "lodash": "^4.17.15",
     "optimist": "^0.6.1",
-    "prettier": "^1.19.1",
-    "tslint": "^5.20.1",
+    "prettier": "^2.0.0",
+    "tslint": "^6.1.0",
```

Comment:
Anything that affects us?

Reply:
Doesn't seem so, was just required for ts 3.8.